modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -290,11 +314,19 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
           startScrollWidth = rootElement.scrollWidth;
           startScrollHeight = rootElement.scrollHeight;
 
-          selectedElmGhost = selectedElm.cloneNode(true);
+          resizeBackdrop = dom.add(rootElement, 'div', { class: 'mce-resize-backdrop' });
+          dom.setStyles(resizeBackdrop, {
+            position: 'fixed',
+            left: '0',
+            top: '0',
+            width: '100%',
+            height: '100%'
+          });
+
+          selectedElmGhost = createGhostElement(selectedElm);
           dom.addClass(selectedElmGhost, 'mce-clonedresizable');
           dom.setAttrib(selectedElmGhost, 'data-mce-bogus', 'all');
-          selectedElmGhost.contentEditable = false; // Hides IE move layer cursor
-          selectedElmGhost.unSelectabe = true;
```

Comment:
This had a typo, so it was doing nothing and `unselectable` isn't a valid property anymore (was in old IE versions) as such I just removed it.